elytron/common/src/main/java/org/wildfly/extension/elytron_common/AcmeAccountService.java

```diff
@@ -16,9 +16,9 @@
  * limitations under the License.
  */
 
-package org.wildfly.extension.elytron;
+package org.wildfly.extension.elytron_common;
```

Comment:
To be consistent with other similar package names, just wondering if this should be:

`org.wildfly.extension.elytron.common`

Reply:
Yeah that should be fine. I think there used to be issues with having the paths nested, but those shouldn't be present anymore.